SYCL/CMakeLists.txt

```diff
@@ -2,6 +2,9 @@ if( NOT OpenCL_LIBRARY )
   find_package(OpenCL)
 endif()
 
+find_package(Threads REQUIRED)
+set(SYCL_THREADS_LIB ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
You should also propagate the value in lit.cfg.py and lit.site.cfg.cpy.in.
Please verify your change before submition

Reply:
The value is already there. I checked it again.
https://github.com/intel/llvm-test-suite/blob/intel/SYCL/lit.cfg.py#L113
https://github.com/intel/llvm-test-suite/blob/intel/SYCL/lit.site.cfg.py.in#L25

So only changes in SYCL/CMakeLists.txt are needed for SYCL_THREADS_LIB to work with threads.